src/components/graphs/bargraph/BarGraph.js

```diff
@@ -1,15 +1,57 @@
 import React, { useState, useEffect } from 'react';
-import { Bar } from 'react-chartjs-2';
-import { createDataSet } from '../assets/bargraphAssets';
+import { Bar, HorizontalBar } from 'react-chartjs-2';
+
+import './BarGraph.less';
+
+const graphOptions = {
+  responsive: true,
+  maintainAspectRatio: false,
+  legend: {
+    display: false,
+    position: 'left',
+    labels: {
+      boxWidth: 0,
+    },
+  },
+};
+
+const horizOptions = {
```

Comment:
Forgot to mention - love that you introduced a horizontal bar graph per our conversation about better means to display this comprehensive data set!